.github/workflows/ccache-build.yaml

```diff
@@ -0,0 +1,75 @@
+---
+name: DNF 5 Build
+on:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  build-dnf5:
+    name: CMake Build
+    runs-on: ubuntu-latest
+    container:
+      image: fedora:rawhide
+    strategy:
+      fail-fast: false  # don't fail all matrix jobs if one of them fails
+      matrix:
+        compiler: ['gcc', clang]  # gcc is the default
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      # this step needs to be incorporated in our container that we use for testing
+      - name: Install dependencies
+        run: |
+          dnf install -y ccache clang && dnf builddep -y ./dnf5.spec
```

Comment:
@m-blaha about this, would you rather update our current container that we use for CI or should we create a separate image for building?

Reply:
I'd prefer to keep one container.

Reply:
also, the COPR secret needs to be fixed. I'll do those two fixes before merging